### PR TITLE
v6.8.0 update

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

File: src/data/en/api.tsx [hunk 1]
```diff
@@ -414,6 +414,7 @@ export default {
         </p>
       </>
     ),
+    isSubmitSuccessful: <p>Indicate the form is successfully submitted.</p>,
```
Comment (pmaier983) [inline src/data/en/api.tsx:417]:
```suggestion
    isSubmitSuccessful: <p>Indicate the form was successfully submitted.</p>,
```

**Review by pmaier983 — DISMISSED** [review]
Looks Good 😄

**Comment by bluebill1049** [comment]
Thanks for the review @pmaier983

**Review by pmaier983 — APPROVED** [review]
Awesome!

**Pull request merged**